tests/unit/scene/test_clipboard.py

```diff
@@ -42,7 +42,10 @@ def setup(self, mocker: MockerFixture):
             edge.destination_socket.id = dummy_edges_links[i][1]
 
         self.scene.sortedSelectedItems.return_value = self.blocks, self.edges
-        self.clipboard = SceneClipboard(self.scene)
+        self.clipboard = BlocksClipboard()
```

Comment:
Here you could pass scene to the constructor, and thus remove scene from the inputs of BlocksClipboard methods.

Reply:
Doing this could make the reader think that the clipboard is tied to one scene, which it is not. The clipboard is global (otherwise, you wouldn't be able to copy and paste from one scene to another).
I made this explicit by removing any "self.scene", and replacing it by passing the scene parameter to the private functions.